src/Utils/request/api.tsx

```
@@ -16,6 +16,13 @@
 
 import { PaginatedResponse } from "@/Utils/request/types";
 import { AppointmentPatientRegister } from "@/pages/Patient/Utils";
+import {
```

ESLint

[error] Insert `·MFAAuthenticationToken·}·from·"@/types/auth/otp";⏎import·{`

(prettier/prettier)